core/src/test/resources/application_test.conf

```diff
@@ -48,7 +48,7 @@ filodb {
     create-tables-enabled = true
     num-token-range-splits-for-scans = 2
     index-scan-parallelism-per-shard = 2
-    part-keys-v2-table-enabled = false
+    part-keys-v2-table-enabled = true
```

Comment:
Should the corresponding [config](https://github.com/filodb/FiloDB/blob/1cc39d9baef00610f9e72aad03e86f0c967e17d0/core/src/main/resources/filodb-defaults.conf#L548) in filo-defaults also be updated since all tests going forward will use v2 tables?

Reply:
I don't want to change the defaults, the reason is that in that case the production clusters will pick up the change but it should be applied only to NEW clusers. In other words, the partitions that are already built will stay on the good old table structure as we don't have the way to transfer the data.